pkg/ring/kv/memberlist/memberlist_client.go

```diff
@@ -134,6 +135,7 @@ type KVConfig struct {
 	DeadNodeReclaimTime time.Duration `yaml:"dead_node_reclaim_time"`
 
 	// List of members to join
+	JoinService      string              `yaml:"join_service"`
```

Comment:
Would it make sense to use this as a boolean flag to enable SRV records lookup for names in join_members?